src/main/java/club/sk1er/patcher/config/PatcherConfig.java

```diff
@@ -75,7 +75,7 @@ public class PatcherConfig extends Vigilant {
         type = PropertyType.SELECTOR, name = "Keyboard Layout",
         description = "The layout of your keyboard, used to fix input bugs accordingly.",
         category = "Bug Fixes", subcategory = "Linux",
-        options = {"QWERTY", "AZERTY"}
+        options = {"QWERTY", "FR AZERTY", "BE AZERTY"}
```

Comment:
```suggestion
        options = {"QWERTY", "BE AZERTY", "FR AZERTY"}
```
Since BE AZERTY was what the old implementation used, it should come first so it continues to work correctly for users with that option selected. Obviously the order in the map needs to be changed as well.